spec.md

```diff
@@ -2232,6 +2244,17 @@ The following table shows what the Plugin SHOULD return when receiving a second
 | MULTI_NODE     | OK (idempotent) | ALREADY_EXISTS | OK                  | OK                 |
 | Non MULTI_NODE | OK (idempotent) | ALREADY_EXISTS | FAILED_PRECONDITION | FAILED_PRECONDITION|
 
+NOTE: If the Plugin supports the `SINGLE_NODE_MULTI_WRITER` capability, use the following table instead for what the Plugin SHOULD return when receiving a second `NodePublishVolume` on the same volume on the same node:
+
```

Comment:
So I think we might have a problem here - can you query control-plane capabilities from node service?

Reply:
The node service has an equivalent capability bit that is used for determining which access mode to supply for node operations that take `VolumeCapability`.